spec/fixtures/iso-pubid-languages.txt

```diff
@@ -31,7 +31,7 @@ ISO 6091:2010(ru)
 ISO 8081:2021(fr)
 ISO 8637:2010/Amd.1:2013(ru)
 ISO/ASTM DIS 52920(fr)
-ISO/CEI 17025:2005/Cor.1:2006(F)
+ISO/IEC 17025:2005/Cor.1:2006(F)
```

Comment:
This is actually correct -- in the French PubID, "IEC" is "CEI". But maybe you can move it to the implementation of #6 .